extension/popup/popupView.jsx

```diff
@@ -633,11 +631,11 @@ const SearchResultsContent = ({
   return (
     <React.Fragment>
       <TextDisplay displayText={displayText} />
-      <div id="search-results">{renderCard()}</div>
+      <div id="search-results">{ transcript.toLowerCase() == 'Hello' ? HelloCard() : renderCard()}</div>
       <div id="search-footer">
         <IntentFeedback
           onSubmitFeedback={onSubmitFeedback}
-          eduText={card && card.answer ? card.answer.eduText : null}
+          eduText={card && card.answer ? card.answer.edutext : null}
```

Comment:
Did you intend to change the case of `eduText`?

Reply:
Hello @ianb was testing out something? That was a mistake, already fixed it on my local machine. Sorry about this.